src/observers/environment.observer.ts

```diff
@@ -37,6 +37,7 @@ export class EnvironmentObserver implements LifeCycleObserver {
     'INFLUX_ORG',
     'ARCHIVAL_CHAINS',
     'BLOCKED_ADDRESSES_URL',
+    'RATE_LIMITER_URL',
```

Comment:
is this truly required? should rate limiter be on by default or should it be a toggle?

Reply:
This allows:

1. Configuration of new endpoint for `rate-limiter` if necessary
2. Toggling `rate-limiter` on/off if any potentially urgent issues arise.

Reply:
Thank you for the review. Fixed.